ci/Dockerfile.circle_build

```diff
@@ -1,10 +1,12 @@
 # To build and deploy, from this directory:
 # $ docker build . -f Dockerfile.circle_build -t us.gcr.io/broad-dsp-gcr-public/workbench:buildimage-X.Y.Z .
 # Note: M1 Macs may need to add `--platform linux/x86_64` after `docker build`
+
 # Test out the new image with:
 # $ docker run -it us.gcr.io/broad-dsp-gcr-public/workbench:buildimage-X.Y.Z /bin/bash
+
 # Update all mentions of us.gcr.io/broad-dsp-gcr-public/workbench:buildimage-X.Y.Z in
-# .circleci/config.yml by incrementing the numbers. Using the new values, run:
+# .circleci/config.yml by incrementing the numbers. Using the new values, run (while logged in to a @broadinstitute.org account):
```

Comment:
There are security concerns with opening this up to non-Broadies, so we should look for an alternative. https://pmi-engteam.slack.com/archives/CHRN2R51N/p1708633092466819